src/legacy/core_plugins/vis_type_timelion/public/index.scss

```diff
@@ -0,0 +1,5 @@
+@import 'src/legacy/ui/public/styles/styling_constants';
+
+@import './timelion_vis';
+@import './timelion_editor';
```

Comment:
I would assume the editor styles need to stay with the actual Timelion app components in `core_plugins/timelion`

Reply:
`timelion_editor` styles are related to Default Editor on the Timelion visualization page


<img width="300" src="https://user-images.githubusercontent.com/11524637/72341931-90e73d00-36dc-11ea-8f92-c2e6ce89130e.png" />

So I moved them to `vis_type_timelion`.

Reply:
Ahh ok I was mistaken, I thought that `vis_type_timelion` just includes what it needs to **render** a timelion vis and not the pieces needed to **create** one. I see now that the other `vis_type_...` folders include the editor styles.